RUFAS/routines/animal/ration/animal_requirements.py

```diff
@@ -1303,14 +1309,13 @@ def calculate_NRC_calcium_requirements(
         if animal_type in [AnimalType.LAC_COW]:
             # [A.Cow.C.1]
             # Calcium maintenance requirement (g)
-            if lactating:
-                Ca_maint = 0.031 * body_weight + 0.08 * (body_weight / 100)
-            else:
-                Ca_maint = 0.0154 * body_weight + 0.08 * (body_weight / 100)
-        elif animal_type in [AnimalType.HEIFER_I, AnimalType.HEIFER_II, AnimalType.HEIFER_III, AnimalType.DRY_COW]:
+            Ca_maint = 0.031 * body_weight + 0.08 * (body_weight / 100)
+        elif animal_type in [AnimalType.DRY_COW]:
+            Ca_maint = 0.0154 * body_weight + 0.08 * (body_weight / 100)
+        elif animal_type in [AnimalType.HEIFER_I, AnimalType.HEIFER_II, AnimalType.HEIFER_III]:
             # [A.Heifer.C.1]
             # Calcium maintenance requirement (g)
```

Comment:
```suggestion
        calcium_weight_factor = 0.0153
        if animal_type in [AnimalType.LAC_COW]:
            calcium_weight_factor = 0.031
        Ca_maint = calcium_weight_factor * body_weight + 0.08 * (body_weight / 100)
```
I think this maintains correct behavior, however it may not be appropriate if the calcium maintenance calculations are expected to diverge for dry cows and heifers at some point.

Reply:
I'll ask Edward about this, as it may be important to maintain the equations in the form they're cited in the reference material. There may indeed be cases in which these can be modified in a more complex manner, if need be.